.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+# Purpose:
+# Automatically bumps the project's patch version bi-weekly on Fridays.
+#
+# Details:
+# - Triggered at 00:00 UTC every Friday; runs on even weeks of the year.
+# - Sets up the environment by checking out the repo and setting up Node.js.
+# - Bumps patch version using `npm version patch`, then creates a new branch 'release/x.x.x'.
+# - Pushes changes and creates a PR to `main` using GitHub CLI.
+# - Can also be triggered manually via `workflow_dispatch`.
+name: Version Bump
+
+on:
+  workflow_dispatch: # Allow to manually trigger the workflow
+  schedule:
+    - cron: "0 0 * * 5" # At 00:00 UTC every Friday
+
+jobs:
+  version-bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Since cronjob syntax doesn't support bi-weekly schedule, we need to check if it's an even week or not
+      - name: Check if it's an even week
+        id: check-week
+        run: |
+          WEEK_NUM=$(date +'%V')
+          if [ $((WEEK_NUM % 2)) -eq 0 ]; then
+            echo "RUN_JOB=true" >> $GITHUB_ENV
+          else
+            echo "RUN_JOB=false" >> $GITHUB_ENV
+          fi
+
+      - name: Check out the repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+
+      - name: Configure Git
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+
+      - name: Bump patch version
+        id: version-bump
```

Comment:
Same as above + this id duplicates Job name.

Reply:
fixed!